### PR TITLE
Moving items from PR template into Contributor policy

### PR DESCRIPTION
**Pull request by asirota**
# Pull Request

## What changed?

This change is in relation to issue #11 

The sentences serve to confuse and potentially scare any contributors. Removing these 2 sentences will lighten the tone of the PR template.

## Why did it change?

Suggestion from the AspireUpdate team...

## Did you fix any specific issues?

#11 

## CERTIFICATION

By opening this pull request, I do agree to abide by the [CODE OF CONDUCT](https://github.com/aspirepress/.github/CODE_OF_CONDUCT.md) and be bound by the terms of the [Contribution Guidelines](https://github.com/aspirepress/.github/CONTRIBUTING.md) in effect on the date and time of my contribution as proven by the revision information in GitHub. I also agree that any previous contributions shall be deemed subject to the terms of the ersion in effect on the date and time of this pull request, or any future revisions for pull requests I may submit.

Further, I certify that this work is my own, is original, does not violate the intellectual property of any other person or entity, and I am not violating any license agreements or contracts I have with any person or entity. Finally, I agree that this code may be licensed under any license deemed appropriate by AspirePress, including but not limited to open source, closed source, proprietary or custom licenses, and that such license terms neither violate my rights or my copyright to this code.

### CODE REVIEW

**Comment by asirota** [comment]
I'll add @costdev when he joins as a contributor to the AP project on GitHub.

**Review by namithj — APPROVED** [review]

**Review by costdev — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -76,6 +76,9 @@ Your license of your contributions to AspirePress relieves us of the right to re
 Note that we may "dual license" any project at any time. A "dual license" may require certain users to use a particular license based on conditions within that license; all others may use the original or new (but similarly restrictive) license. The "dual license" model does not in any way constitute a reduction in rights or freedoms granted users, and any user that wishes to remain on an older version under the original license may do so. We will not retroactively change license terms.
 
 
+When creating a pull requests or committing changes to any AspirePress repository, contributors certify that their contributions are their own, are original and do not violate the intellectual property of any other person or entity. The author also certifies that they are not violating any license agreements or contracts they have with any person or entity. Contributors also agree that any contributioons may be licensed under any license deemed appropriate by AspirePress, including but not limited to open source, closed source, proprietary or custom licenses, and that such license terms neither violate their rights or their copyright to the contributions.
```
[the remark this answers is not in the document]
Comment (asirota) [reply CONTRIBUTING.md:79]: @costdev can you please go the issue #11 and state your concern here and I'll respond there.

**Comment by chuckadams** [comment]
Less boilerplate good, but maybe we should at keep a single sentence like "By submitting this PR I acknowledge to abide by the terms of the Code of Conduct and the Contributor Guidelines." (with appropriate links).  Just a HR separating it, no need for a big CERTIFICATION header even.

**Comment by asirota** [comment]
Fixed the typos for the hyperlinks to code of conduct and contributing files.

**Review by toderash — APPROVED** [review]

**Pull request merged**